tests/integration/integration_steps/fabric_steps.py

```diff
@@ -12,10 +16,25 @@ def read_deltalake_timeseries(timeseries_path:str, credential: DefaultAzureCrede
     df = delta_table.to_pandas()
     return df
 
+def prepare_lakehouse_dataframe_for_comparison(dataframe, external_id):
+    dataframe = dataframe.loc[dataframe["externalId"] == external_id]
+    if dataframe[TIMESTAMP_COLUMN].dt.tz is None:
+        local_tz = tz.tzlocal()
+        dataframe[TIMESTAMP_COLUMN] = dataframe[TIMESTAMP_COLUMN].dt.tz_localize(local_tz)
+    dataframe[TIMESTAMP_COLUMN] = dataframe[TIMESTAMP_COLUMN].dt.tz_convert('UTC')
+    dataframe[TIMESTAMP_COLUMN] = dataframe[TIMESTAMP_COLUMN].dt.round('s')
+    return dataframe
```

Comment:
Line 23 is localizing the dataframe because it's timezone naive when it comes from the Lakehouse.  At least in my Lakehouse, the timestamps are in Central time.

Reply:
interesting well it works either way and think this is good enough to compare the timestamps